### DISCUSSION TITLE
Example Project?

### DISCUSSION THREAD

**Discussion by sirwesley** [typedorm/typedorm#181] [Q&A] [answered]
Looking for recommendations for an example project that pulls all the pieces together?  

Currently, have typedorm working in a local environment with NestJS & DynamoDb, however, trying to make the jump to production with Serverless AWS Lambda is becoming a headache.  

The serverless app complains about:
Error: No such connection with name "default" exists 
at ConnectionManager.get (/var/task/node_modules/@typedorm/core/src/classes/connection/connection-manager.js:21:23)
at getEntityManager (/var/task/node_modules/@typedorm/core/public-api.js:60:35)

Not sure when the 'default' connection should get instantiated/configured  (before or after the app bootstraps?).     

Having a good starter project for reference would be a great help. 
Any help would be greatly appreciated. 

-Wesley

**Comment by whimzyLive** [comment, marked as answer]
Hi @sirwesley 

It really is a good idea to have an example project. I will see if I can get something up within the next week or so. 
Ideally, you will need to execute TypeDORM `createConnection` before the nestjs `init`.

Are you able to provide a quick repro/project where I can help configure it? 

On a side note, if it worked locally then it should just work the same way in the cloud unless you have some other workflow setup for lambda.
